theano/configdefaults.py

```diff
@@ -1238,21 +1252,20 @@ def default_blas_ldflags():
                                           "mk2_rt"]])
 
         # Anaconda
-        if "Anaconda" in sys.version or "Continuum" in sys.version:
-            # If the "mkl-service" conda package (available
-            # through Python package "mkl") is installed and
+        if "Continuum Analytics" in sys.version:
```

Comment:
Is this also true for older Anaconda version? I'm not opposed to only support current Anaconda, but if we do so, I would add that in the documentation so user don't get surprised.